modules/ROOT/pages/index-policies.adoc

```diff
@@ -40,10 +57,17 @@ This layer of security provides the same control as any API Manager policy offer
 * OAuth
 * Authorization and Accounting (AAA)
 
-You must have Runtime Fabric and Security Edge Policies entitlements enabled for your Anypoint Platform account, in addition to API Manager and Analytics.
+== Prerequisites
```

Comment:
I must be missing something...how did we get from the topic of edge security policies to the prereqs for configuring and using them (where is the task for configuring? using?); to the description list of policy names; to prerequisites for tokenization?

Reply:
C&P error. You aren't missing anything.